<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefDialog</class>
 <widget class="QDialog" name="DefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Definition</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushApply">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>97</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose Definition File</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboDef"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelDef">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushUpdate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>94</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Update List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDownload">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>94</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Update</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QListView" name="listUpdate">
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
